cookbook/zemismart-rgbw-downlights.rst

```diff
@@ -0,0 +1,196 @@
+Zemismart LED RGBW Downlights
+=============================
+
+The Zemismart LED RGBW Downlight is a tuya based downlight available from various retailers online or from `zemismart.com <https://www.zemismart.com>`__ direct.
+
+.. figure:: images/zemismart-rgbw-downlight.jpg
+    :align: center
+    :width: 50.0%
+
+Originally intended to be used with their companion app once flashed using `tuya-convert <https://github.com/ct-Open-Source/tuya-convert>`__ ESPHome generated
```

Comment:
Is this still up-to-date? I heard tuya-convert no longer works on most devices.

Reply:
I've still been able to use it to flash devices along with others on this thread - https://community.home-assistant.io/t/australia-electrically-certified-hardware/32074/275?u=cryptelli